WeakAuras/RegionTypes/DynamicGroup.lua

```diff
@@ -416,8 +416,8 @@ end
 
 local growers = {
   LEFT = function(data)
-    local stagger = -(data.stagger or 0)
-    local space = data.space or 0
+    local stagger = -(data.stagger or 0) * Private.PixelMult
```

Comment:
This can't be backwards compatible.

There are a few options:
a) Make a global WeakAuras wide option that enables it. So basically a user has to opt into it.
b) Make this a per aura setting (Or a per group?)
c) Don't multiply but align

Things to consider are:
* Pixel-Perfect auras (whether that's global or per aura setting) are harder to share. Because of the way uiscale uses the same ui scale, the auras "appear" to be similar size regardless of actual screen resolution. (Which is only true for users that don't set the uiscale)
* There's a ticket for it: https://github.com/WeakAuras/WeakAuras2/issues/236
* And Blizzard has some utility functions, see https://github.com/tomrus88/BlizzardInterfaceCode/blob/master/Interface/SharedXML/PixelUtil.lua